gc/gc-base/src/main/java/org/projectnessie/gc/base/DistributedIdentifyContents.java

```diff
@@ -65,21 +71,27 @@ public DistributedIdentifyContents(SparkSession session, GCParams gcParams) {
    *
    * @param liveContentsBloomFilterMap live contents bloom filter per content id.
    * @param references list of all the references (JSON serialized) to walk (live and dead)
-   * @return {@link IdentifiedResult} object.
+   * @return current run id of the completed gc task
    */
-  public IdentifiedResult getIdentifiedResults(
+  public String identifyExpiredContents(
       Map<String, ContentBloomFilter> liveContentsBloomFilterMap, List<String> references) {
-
+    String runId = UUID.randomUUID().toString();
+    Timestamp startedAt = Timestamp.from(Instant.now());
```

Comment:
nit: allow injection of `Instant.now()` for testing

Reply:
over all (Instant like commit time, author time etc) in GC cannot be mocked as Nessie backend runs in quarkus server and long back faced many problems to mock the time in quarkus server. 

Plus this is not an input argument, this becomes output argument. Mocking is not necessary.